Sources/ViewInspector/Inspector.swift

```diff
@@ -1,18 +1,125 @@
 import SwiftUI
 
 @available(iOS 13.0, macOS 10.15, tvOS 13.0, *)
-public struct Inspector { }
+public struct Inspector {
+
+    /// Removes the "(unknown context at <memory_address>)" portion of a type name.
+    /// Calls to this method are memoized and retained for the lifetime of the program.
+    /// - Parameter typeName: The raw type name. (e.g. ``SomeTypeName.(unknown context at $138b3290c).SomePropertyName``)
+    /// - Returns: The sanitized type name. (e.g. ``SomeTypeName.SomePropertyName``)
```

Comment:
```suggestion
    /// - Parameter typeName: The raw type name. (e.g. `SomeTypeName.(unknown context at $138b3290c).SomePropertyName`)
    /// - Returns: The sanitized type name. (e.g. `SomeTypeName.SomePropertyName`)
```

Nit: My understanding is that the double backticks are used to create links to other types when documentation comments are displayed in Xcode. Since these are example types, I think single backticks are more appropriate.

Reply:
Makes sense! I'll update to single backticks! 👍